i18n/en/docusaurus-plugin-content-docs/version-5.0/02-快速入门/02quickstart.md

```diff
@@ -1,81 +1,79 @@
-# 快速开始
+# Quickstart
 
-这一节介绍如何快速部署一个单节点单副本 RocketMQ 服务，并完成简单的消息收发。
+This section will introduce the method of quickly building and deploying a single-Master RocketMQ cluster to complete simple message sending and receiving.
 
-:::tip 系统要求
+:::tip SYSTEM REQUIREMENT
 
-1. 64位操作系统，推荐 Linux/Unix/macOS
-2. 64位 JDK 1.8+
+1. 64-bit OS，Linux/Unix/macOS is recommended
+2. 64-bit JDK 1.8+
 
 :::
 
-## 1.下载安装Apache RocketMQ
+## 1.Get Apache RocketMQ
 
-:::tip RocketMQ下载
+:::tip Download RocketMQ
 
-RocketMQ 的安装包分为两种，二进制包和源码包。 点击[这里](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-source-release.zip) 下载 Apache RocketMQ 4.9.4的源码包。你也可以从[这里](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-bin-release.zip) 下载到二进制包。二进制包是已经编译完成后可以直接运行的，源码包是需要编译后运行的，
+RocketMQ's installation is divided into two types: binary and source.  Click [here](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-source-release.zip) to download Apache RocketMQ 4.9.4 source package. You can also get from [here.](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-bin-release.zip) The binary package can be run directly since it has been compiled, and the source package needs to be compiled and run.
 
 :::
 
-这里以在Linux环境下利用社区5.0.0的源码包为例，介绍RocketMQ安装过程。
+The following instruction takes the application of RocketMQ 4.9.4 source package in Linux environment as an example in order to introduce the installation process of RocketMQ.
 
-解压4.9.4的源码包并编译构建二进制可执行文件
+Extract the source package of RocketMQ 4.9.4, then compile and build the binary executables:
 
 ```shell
```

Comment:
Why does 'Quick Start' contain instructions to build from the source? It would be better to launch a docker or deploy a Helm chart.